templates/project/_modalSearch.html.twig

```diff
@@ -0,0 +1,32 @@
+<button type="button" class="btn btn-primary" data-bs-toggle="modal" data-bs-target="#searchContributorModal">
+    Search a contributor
+</button>
+
+<div class="modal fade" id="searchContributorModal" tabindex="-1"aria-hidden="true">
+    <div class="modal-dialog">
+        <div class="modal-content">
+            <div class="modal-header">
+                <h5 class="modal-title">Search a contributor</h5>
+                <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+            </div>
+            <div class="modal-body">
+                <form id="search-form" name="search-contributor" action="{{ path('ajax_search_contributor') }}"
+                      data-ajax-url="{{ path('ajax_search_contributor') }}">
```

Comment:
What is the point of having this url in action AND in data-ajax-url ?

Reply:
I wouldn't know what to answer your question. but now we got only one path and is work .